frontend/test/__support__/server-mocks/collection.ts

```diff
@@ -104,46 +104,44 @@ function setupCollectionWithErrorById({
   error: string;
   status?: number;
 }) {
-  fetchMock.get(/api\/collection\/\d+/, () => {
+  fetchMock.get(/api\/collection\/\d+|personal|root/, () => {
     return {
       body: error,
       status,
     };
   });
 }
 
-export function setupRootCollectionItemsEndpoint({
-  collections = [],
-  dashboards = [],
-}: {
-  collections: Collection[];
-  dashboards: Dashboard[];
-}) {
-  fetchMock.get("path:/api/collection/root/items", () => {
-    const rootDashboards = dashboards.filter(
-      dashboard => dashboard.collection_id === null,
-    );
-    const rootCollections = collections.filter(
-      collection => collection.location !== "/",
-    );
-    const data = [...rootDashboards, ...rootCollections];
-
-    return {
-      total: data.length,
-      data,
-    };
-  });
-}
-
 export function setupCollectionItemsEndpoint({
+  collections = [],
```

Comment:
Could you accept a single collection like search helper? It won't be easy/possible to use this helper with multiple collection anyway - because it can't be the case that the same item is within 2+ collections